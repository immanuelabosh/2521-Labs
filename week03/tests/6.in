-1
-5
-6
-7
-12
-25
-14
-75
-63
-52
-35
-40