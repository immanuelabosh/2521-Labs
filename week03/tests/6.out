Tree:
             -1
             /
            -5
           /
          -6
         /
        -7
       /
     -12
     /
   -25
   / \
  /   \
-75   -14
  \
  -63
    \
    -52
      \
      -35
      /
    -40

#nodes = 12,  #leaves = 2
Infix  : -75 -63 -52 -40 -35 -25 -14 -12 -7 -6 -5 -1 
Prefix : -1 -5 -6 -7 -12 -25 -75 -63 -52 -35 -40 -14 
Postfix: -40 -35 -52 -63 -75 -14 -25 -12 -7 -6 -5 -1 
ByLevel: -1 -5 -6 -7 -12 -25 -75 -14 -63 -52 -35 -40 
